typescript/examples/langchain-privy-chatbot/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Note: When running format, it failed on this file due to the lack of .prettierrc. Added it to pass.

If this should be reverted lmk